index.js

```diff
@@ -0,0 +1,59 @@
+let form = document.getElementById("new-task-form");
+let input = document.getElementById("new-task-input");
+let message = document.getElementById("message");
+let posts = document.getElementById("tasks");
+
+form.addEventListener("submit", (e) => {
+  e.preventDefault();
+  formValidation();
+});
+let formValidation = () => {
+  if (input.value === "") {
+    message.innerHTML = "Input field can not be blank...";
+  } else {
+    message.innerHTML = "";
+    acceptData();
+  }
+};
+let data = [];
+
+let acceptData = () => {
+  data.push({
+    text: input.value,
+  });
+  localStorage.setItem("data", JSON.stringify(data));
+  createPost();
+};
+let createPost = () => {
+  posts.innerHTML = "";
+  data.map((x, y) => {
+    return (posts.innerHTML += ` <div id=${y}><div id="tasks">
```

Comment:
you are missing one closing div tag, please indent your code so it's easier to notice mistakes like this